instrument/D11B_Parameters.xml

```diff
@@ -23,6 +23,78 @@
       <parameter name="y-pixel-size" type="number">
         <value val="8"/>
       </parameter>
+      <!-- These parameters are used to define headers and entries for GenerateLogbook algorithm -->
+      <parameter name="logbook_default_parameters" type="string">
+	<value val="SampleDescription:/entry0/sample_description,
+		    TotalTime:/entry0/duration,
```

Comment:
I agree this is not a pretty solution, intended for readability and ease of future modifications. I agree that your solution does look better and XML-like (but one should use slashes instead of backslashes), however it doesn't seem to work with Mantid. Unfortunately, only the final defined value is registered, and the rest is lost.

Reply:
OK I see. So the Mantid parameter map functionality only permits\expects a single value. I guess you could embed the xml values list inside a single val attribute using something like:
```
<value val="&lt;logvalue=&quot;wavelength&quot; path=&quot;/entry0/D7/monochromator/wavelength&quot;\&gt;"\>
```
...but this is perhaps less user friendly than the approach you've taken. So what you have done looks fine.

Have you got the divide operator to work OK? It seems like it will be difficult for the code to correctly identify the "//" string inside a formula when the two paths also start with a "/" character. Is this the syntax you would use for a divide?

/entry0/D7/POL/actual_state///entry0/D7/POL/actual_stateB1B2

Reply:
Yes indeed, it seems that only one value can be assigned to a parameter. It's good to know there exist a workaround should one require it.

I have tested the behaviour of the operations and regexes in a separate small-scale script  to verify different input types and operations. I can send it your way if you would like to play with it outside of Mantid for more flexibility. The division is going to be performed as you showed, when 3 subsequent slashes are present, which is not the most appealing solution but it does work.

Reply:
That's fine, no need to send me the script. I wasn't sure if the code would use the 1st\2nd slash or the 2nd\3rd in a group of three slashes as the divide operator but I've just tried it and it looks like you've sorted all of this out.
You could possibly add a space around the "//" when building up the header at ~L190 of GenerateLogBook.py so it's clear that the operator is unrelated to the slashes in the path but this is "nice to have". Perhaps let's agree on whether there are any other changes first...